### DISCUSSION TITLE
What is the best way to post-process the results of Openradioss with an open-source software?

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#1480] [Q&A] [answered]
Hi, I'm wondering which is the best way to see the results of a simulation with Openradioss, h3d or anim?
Is there a fastest way to convert those files for paraview on windows?

**Comment by sebastienVilleneuve** [comment, marked as answer]
Hi @FGarelli 

You can output anim files & convert to vtk 
you can also convert T01 file to csv file 
https://github.com/OpenRadioss/OpenRadioss/tree/main/tools

Both vtk & csv format are supported by Paraview

Regards,
Sébastien
